moabb/analysis/meta_analysis.py

```diff
@@ -121,15 +121,15 @@ def _pairedttest_random(data, nperms):
     pvals: ndarray of shape (n_pipelines, n_pipelines)
         array of pvalues
     """
-    out = np.ones((data.shape[1], data.shape[1]))
+    out = np.zeros((data.shape[1], data.shape[1]))
```

Comment:
In the case of exhaustive, this is not necessary.
But in the case of random, the initial statistic is not necessarily among those found within the permutation distribution.